core/test/scripts/Voting.js

```diff
@@ -22,6 +22,46 @@ class MockNotifier {
   }
 }
 
+const TEST_IDENTIFIERS = {
+  test: {
+    key: web3.utils.utf8ToHex("test"),
+    expectedPrice: web3.utils.toWei("1.5")
+  },
+  "test-bad-reveal": {
+    key: web3.utils.utf8ToHex("test-bad-reveal")
+  },
+  "test-overlapping1": {
+    key: web3.utils.utf8ToHex("test-overlapping1"),
+    expectedPrice: web3.utils.toWei("1.5")
+  },
+  "test-overlapping2": {
+    key: web3.utils.utf8ToHex("test-overlapping2"),
+    expectedPrice: web3.utils.toWei("1.5")
+  },
+  TSLA: {
+    key: web3.utils.utf8ToHex("TSLA"),
+    hardcodedTimestamp: "1573513368",
+    expectedPrice: web3.utils.toWei("345.07")
+  },
+  "Custom Index (1)": {
+    key: web3.utils.utf8ToHex("Custom Index (1)"),
+    expectedPrice: web3.utils.toWei("1")
+  },
+  "Custom Index (100)": {
+    key: web3.utils.utf8ToHex("Custom Index (100)"),
+    expectedPrice: web3.utils.toWei("100")
+  },
+  "0.5": {
+    key: web3.utils.utf8ToHex("0.5"),
+    expectedPrice: web3.utils.toWei("0.5")
+  }
+};
+
+const getRandomTimestampInPast = () => {
```

Comment:
I think a function like this could be really useful in many places! What do you think about moving it up to `common/Random.js`?

Reply:
for now I removed this method in favor of the contract's `getCurrentTimestamp()` method, but next time we need an offchain-generated random timestamp in the past we should add to `common/Random.js`